eng/terraform/main.tf

```diff
@@ -0,0 +1,78 @@
+terraform {
+  required_providers {
+    azurerm = {
+      source  = "hashicorp/azurerm"
+      version = "=3.0.0"
+    }
+  }
+}
+
+# Configure the Microsoft Azure Provider
+provider "azurerm" {
+  features {}
+  skip_provider_registration = "true"
+}
+####
+locals {
+  
+}
+
+####
+# analytics perf testing VM
+# SqlVirtualMachine | MicrosoftSQLServer - SQL2017-WS2016 - SQLDEV | Standard_DS11_v2 | South Central US
```

Comment:
We should be building all of our servers with SQL Server 2019 on Windows Server 2022 at this point.

Reply:
this was again me recreating what was already there so i will update the remaining resources with the latest images